WordPressAuthenticator/WordPressAuthenticator/Authenticator/WordPressSupportSourceTag.swift

```diff
@@ -13,11 +13,16 @@ public enum WordPressSupportSourceTag {
     case loginUsernamePassword
     case loginWPComPassword
     case wpComCreateSiteCreation
+    case wpComCreateSiteCategory
+    case wpComCreateSiteTheme
     case wpComCreateSiteDomain
     case wpComCreateSiteDetails
     case wpComCreateSiteUsername
     case wpComSignupEmail
     case wpComSignup
     case wpComSignupWaitingForGoogle
     case wpComSignupMagicLink
+    case wpComLogin
+    case wpOrgLogin
+    case inAppFeedback
 }
```

Comment:
This one is potentially problematic, long term. Reasons:

- This class lives in a framework that does not have knowledge of the `Create Sites` flow.
- inAppFeedback is also Client-App Side.

Suggestion!!. I think it's time we converted `WordPressSupportSourceTag` into an extensible enum. This way we get to keep the "Enum Cases" we need, in the authenticator, and the client app can piggyback, if needed.

Will send you a PR in a little bit: #9315

Huge WIN about this approach is that: when the Authenticator is in it's own repository, we'd be able to just add "new WordPressSupportSourceTag cases" without the need to actually edit this file. (Details coming up in that PR).